.github/workflows/ci.yml

```diff
@@ -164,6 +164,9 @@ jobs:
           path: |
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+      - name: Clean local Maven repo
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: rm -rf ~/.m2/repository
```

Comment:
Add a code comment?

The commit changes some `install` into `package`, so worth noting why it's still worthwhile to delete `~/.m2/repository`.

Also, why ``~/.m2/repository`` and not eg `~/.m2/repository/io/trino` ?

Reply:
This removes the whole local repo to avoid creating a cache entry from this job since it might not be the one with the most dependencies.

Reply:
Is it worth a code comment?